tests/cypress/tests-e2e/reportProblem.cy.js

```diff
@@ -0,0 +1,170 @@
+/// <reference types="cypress" />
+
+import { APP_VERSION } from '@/config'
+
+const text =
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.'
+const validEmail = 'this.is.a.valid@email.com'
+
+describe('Testing the report problem form', () => {
+    beforeEach(() => {
+        cy.goToMapView()
+    })
+    context('Report problem button placement', () => {
+        it('should be in the configuration section of the menu on mobile', () => {
+            cy.get('[data-cy="menu-button"]').click()
+            cy.get('[data-cy="menu-settings-section"]').click()
+
+            cy.get('[data-cy="report-problem-button"]').should('be.visible').click()
+            cy.get('[data-cy="report-problem-form"]').should('be.visible')
+        })
+        it('should be in the header as a link on desktop', () => {
+            cy.viewport(1920, 1080)
+
+            cy.get('[data-cy="report-problem-link-button"]').should('be.visible').click()
+            cy.get('[data-cy="report-problem-form"]').should('be.visible')
+        })
+    })
```

Comment:
Generally speaking the number of test case `it()` in cypress are expensive, for the cypress.io cloud we pay per it() and more `it()` are less performant (test duration is longer). So due to this we tend to have fewer `it` in cypress e2e tests. You should merge some of the `it` together. To separate logic in one `it` we use `cy.log()` to mention what will be tested to ease debugging in the case of test failure.
So you can keep the `context` as is but try to merge some of the it together. Note in one `it` you can do several `cy.gotToMapView`

Reply:
Ah ok, I have reduced the number of test case now (now only 5).